src/audio/ThreadedAudioDevice.cpp

```diff
@@ -61,10 +61,13 @@ int ThreadedAudioDevice::startThread()
 	PRINT1("\tThreadedAudioDevice::startThread: starting thread\n");
 	pthread_attr_t attr;
 	pthread_attr_init(&attr);
-	int status = pthread_attr_setschedpolicy(&attr, SCHED_RR);
+	int status;
+#ifndef WASM
```

Comment:
I dislike #ifdefs which are specific to a configuration like this -- can you explain why this has to be #ifdef'd here?

Reply:
I think I was having a linking issue with this call (`pthread_attr_setschedpolicy`). I can't reproduce it with the version of Emscripten I have now, so I'll remove the `#ifndef`.